IceCubesApp/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -169,6 +169,7 @@
 "account.action.unblock" = "Unblock";
 "account.action.mute" = "Mute";
 "account.action.unmute" = "Unmute";
+"account.action.share" = "Share this account";
```

Comment:
All the new key need to be added to every files. You can add the english translation as a placeholder.

Reply:
Hmm it still not on every localizable files

Reply:
Are there ones that aren't named Localizable.strings that I should be looking for? Because

```
❯ find . -name Localizable.strings | wc -l
      14
```

Says there are 14 of them total. And these say I have inserted those placeholders into 14 different files of that name:

```

IceCubesApp on  make_shares_translatable [⇡]
❯ find . -name Localizable.strings -exec grep "account.action.share" {} \; | wc -l
      14

IceCubesApp on  make_shares_translatable [⇡]
❯ find . -name Localizable.strings -exec grep "status.card.share" {} \; | wc -l
      14

IceCubesApp on  make_shares_translatable [⇡]
❯ find . -name Localizable.strings -exec grep "status.card.copy" {} \; | wc -l
      14
```

So I don't understand what I missed.

Reply:
Oh 🤦‍♀️ I missed `git push`

Reply:
I guess you'll have quite a bit of conflict now